examples/jbatch/pom.xml

```diff
@@ -0,0 +1,124 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright (c) 2022 Oracle and/or its affiliates.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>io.helidon.applications</groupId>
+        <artifactId>helidon-mp</artifactId>
+        <version>2.4.3-SNAPSHOT</version>
+        <relativePath>../../applications/mp/pom.xml</relativePath>
+    </parent>
+    <groupId>io.helidon.jbatch</groupId>
+    <artifactId>helidon-jbatch-example</artifactId>
+    <name>jbatch-example</name>
+
+    <properties>
+        <javax.batch-api.version>1.0.1</javax.batch-api.version>
```

Comment:
we use `version.lib....` for defining versions.
@barchetta, @romain-grecourt   - is it OK do define versions this way (e.g. outside of dependencies/pom.xml)?

Reply:
Still fixed to be consistent.